src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  --activeLinkColor: #00acdc;
+  --simpleLinkColor: #000;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  margin: auto 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.img {
+  margin: 10px 50px;
+  position: absolute;
+}
```

Comment:
there is no need to use absolute here, find another way of centring, for example, flexbox